.terraform-version

```diff
@@ -0,0 +1 @@
+0.10.1
```

Comment:
Add a note to README that this module requires TF 0.10.x, though I don't think this is the case here. Or, remove this file?